### ISSUE TITLE
Unable to download trial edition on corporate network

### ISSUE THREAD

**Issue by sdsalyer** [flyway/flywaydb.org#98]
Hi there,

I was attempting to download the trial edition of Flyway for evaluation at my company. I could not seem to download the zip file (click trial button, get an accept license popup, navigate to "thank you" page but no download).

I did some inspection and I think somehow my company's proxy or firewall is blocking the request with  407 and 404 response codes. I'm not sure if there's anything you can do on your end, but I wanted to point this out as a potential issue for you, considering I'm sure you'd like to sell more pro/enterprises licenses, and perhaps others behind a proxy are missing the download as well.

See screenshot for reference. I was able to directly load the `/downloads/flyway-trial-edition` page and got a directory listing. I was also able to download the ZIP file eventually this way (direct URL), but I'm not sure if others might be as curious as myself to dig in there and find it.

Hope this helps!

![image](https://user-images.githubusercontent.com/596870/38633285-25584c18-3d8d-11e8-8bb4-8c8158bf0a0c.png)

**Comment by axelfontaine** [comment]
Hmm that's very odd. Thank you very much for reporting. Maybe this is an old IE thing? Which browser did you use?

**Comment by sdsalyer** [comment]
Apologies: I experienced in both Chrome and IE, which is why I chalked it
up to company proxy or firewall. I was working from home, but on company
VPN. I tried from my desktop PC (same home network, no VPN) and had no
issue.

**Comment by axelfontaine** [comment]
Thanks for the details. Then your initial guess is most likely right. We'll add a "If your download doesn't start, click here" link to handle these cases.

**Comment by sdsalyer** [comment]
Perfect. That was my first instinct to look for such a link. There's no
telling what rules might have blocked it or if others had issue, so that
will nip it in the bud!

**Comment by sdsalyer** [comment]
Hi again,

Today I tried to download the 5.1 trial version and was again unable to do so, but the issue appears to be different. It seems the browser is expecting a document for display instead of a download, despite getting a MIME type of application/zip.

In Chrome the "warning" in the console is: 
**Resource interpreted as Document but transferred with MIME type application/zip: "http://files.flywaydb.org/downloads/flyway-trial-edition/flyway-trial-5.1.0.zip".**

![image](https://user-images.githubusercontent.com/596870/40553112-b6b0a42c-600f-11e8-9493-51e39622431d.png)
The download bar displays as "Failed - Network error"

I attempted in IE as well, and the download appears to begin, but then a message "The download was interrupted"

Let me know if I can provide additional details. Again, I am behind a corporate proxy, so I'm not sure if that is the reason for the issue or not.
